clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiAdapter.groovy

```diff
@@ -282,6 +309,38 @@ class KubernetesClientApiAdapter {
     }
   }
 
+  V1beta1DaemonSet deleteDaemonSetPod(String name, String namespace, V1beta1DaemonSet deployedControllerSet) {
+    exceptionWrapper("statefulSets.create", "Replace Daemon Set ${name}", namespace) {
+      def nodeSelector = new HashMap<String, String>()
+      UUID uuid = UUID.randomUUID()
+
+      nodeSelector.put(uuid.toString(), uuid.toString())
```

Comment:
If you're selecting a random UUID, how is this supposet to work?

Reply:
I believe that it is the way to remove the daemonset pod.  -v=9 of kubectl is taking the sane approach but I just don't know how they generate the uid.

Reply:
I double checked the kubectl.  It was calling PUT to replace the daementset with the following nodeselector value in the request payload.
...
"dnsPolicy": "ClusterFirst",
        "nodeSelector": {
          "80b33179-b9bd-11e7-9768-dca9048ce3c9": "80b3325f-b9bd-11e7-9768-dca9048ce3c9"
        },
...
uuid works for me so far.